api/account.go

```diff
@@ -0,0 +1,42 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+// AccountService is a service that interacts with Account related
+// endpoints from the Lacework Server
+type AccountService struct {
+	client *Client
+}
+
+func (svc *AccountService) OrganizationInfo() (
```

Comment:
Every function inside the Go-Client has to have an action, this was missing:
```suggestion
func (svc *AccountService) GetOrganizationInfo() (
```